airbyte-integrations/connectors/source-bing-ads/unit_tests/test_source.py

```diff
@@ -40,10 +40,6 @@ def test_source_check_connection_failed_invalid_creds(config, logger_mock):
     with patch.object(Accounts, "read_records", return_value=[]):
         connected, reason = SourceBingAds().check_connection(logger_mock, config=config)
         assert connected is False
-        assert (
```

Comment:
Related to https://github.com/airbytehq/airbyte/pull/58595/files#r2054607155, I had some issues with requests-mock. I timeboxed this one as this test will probably be removed during the migration anyway